pandas/tests/extension/base/reshaping.py

```diff
@@ -64,6 +64,11 @@ def test_concat_mixed_dtypes(self, data):
         expected = pd.concat([df1.astype('object'), df2.astype('object')])
         self.assert_frame_equal(result, expected)
 
+        result = pd.concat([df1['A'], df2['A']])
```

Comment:
is this just for axis=1? I would make this a separate test.

Reply:
The bug is just with `axis=0`, the default, which this is testing.

For `axis=1` there's not any up-casting required.

Though I don't see any tests for `concat(..., axis='columns')` with the extension array. Can you add some Joris? concat EA and EA, mix of EA and non-EA, aligned and no aligned. Or should I?

Reply:
Hmm, thought we already had some, but indeed, apparently not. Will add some.